spec/syntax.md

```diff
@@ -97,6 +97,17 @@ Attempting to parse a _message_ that is not _valid_ will result in a _Data Model
 
 A **_<dfn>message</dfn>_** is the complete template for a specific message formatting request.
 
+The purpose of a _message_ is to insert _variable_ values into text
+or to select the most appropriate text according to the value of a _variable_
+(or both).
```

Comment:
Talking about the "purpose" of a message feels really deep, and I'm not sure that I completely agree with this characterisation. Do we need this here, or could we leave it out?

Reply:
We could leave it out.

```suggestion
```